draft-privacypass-rate-limit-tokens.md

```diff
@@ -1071,6 +1073,26 @@ for the Issuer. The Attester needs to support key rotation, but ought to disallo
 changes, which could indicate that an Origin is colluding with an Issuer to try to rotate the key
 for each new Client in order to link the client activity.
 
+## Origin Identification
+
+As stated in {{properties}}, the design of this protocol is such that Attesters cannot
+learn the identity of origins that Clients are accessing. The Origin Name itself is
+encrypted in the request between the Client and the Issuer, so the Attester cannot
+directly learn the value. However, in order to prevent the Attester from inferring the
+value, additional constraints need to be added:
+
+- Each Issuer SHOULD serve tokens a large number of Origins. A one-to-one relationship
+between Origin and Issuer would allow an Attester to infer which Origin is accessed
+simply by observing the Issuer identity.
+
+- Issuers SHOULD NOT return rate-limit values that are specific to Origins, such
+that an Attester can recognize an Origin simply by observing the rate limit. This
+can be mitigated by having many Origins share the same rate-limit value, or by introducing
+variability into return the rate-limit value.
```

Comment:
```suggestion
can be mitigated by having many Origins share the same rate-limit value.
```
This is akin to randomized padding which doesn't work well over repeated samples, so I think we ought to keep just the simple thing.